api/config/types.ts

```diff
@@ -7,7 +7,7 @@ import { AppEnum } from '../src/types/internal';
 
 export enum Environment {
   DEVELOPMENT = 'development',
-  TESTING = 'testing',
+  TEST = 'test',
```

Comment:
Why are we doing this?

Reply:
Because this storybook expects this variable to be set in testing environment